README.md

```diff
@@ -21,21 +21,28 @@ When reusing this work please include a proper attribution:
 
 It is recommended to install Pop_Icons through your system package manager, e.g.:
 
-```
+```sh
 sudo apt update
 sudo apt install pop-icon-theme
 ```
 
 If your OS does not package Pop_Icons, you can install them from git:
 
-```
+```sh
 git clone https://github.com/pop-os/icon-theme pop-icon-theme
 cd pop-icon-theme
 meson build
 sudo ninja -C "build" install
 ```
 
-By default it installs to `/usr/` but you can specify a different directory with a prefix like: `/usr/local` or `$HOME/.local`.
+By default it installs to `/usr/` but you can specify a different directory with a prefix like: `/usr/local` or `$HOME/.local`, like below:
+
+```sh
+git@github.com:pop-os/icon-theme.git
```

Comment:
This is not a full command, it's missing `git clone` at the front. However, it's probably better to just remove the first two commands from this block, and only repeat the `meson` and `ninja` lines since `meson` is where the difference starts.

Reply:
Thanks. Resolved.